app/models/like.rb

```diff
@@ -1,4 +1,9 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_likes_counter
+    post.likes_counter = post.likes.count
+    post.save
+  end
```

Comment:
- [ Optional ] Good Job for using [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new like has been saved to database, then **Kindly I recommend you to make the `update_likes_counter` method private**.